netsim/daemons/bird/bgp.j2

```diff
@@ -49,12 +49,15 @@ filter bgp_export_{{ ntype }} {
 {% endfor %}
 
 {% for n in bgp.neighbors %}
-{%   for af in [ 'ipv4','ipv6' ] if af in n %}
+{%   for af in [ 'ipv4','ipv6' ] if af in n and n[af] is string %}
```

Comment:
I knew this wouldn't end well before I even started the tests. How do you configure unnumbered BGP neighbors?

Reply:
Removed BGP IPv6 LLA support